app/src/main/java/com/palone/planahead/ui/theme/Theme.kt

```diff
@@ -1,10 +1,68 @@
-package com.example.compose
+package com.palone.planahead.ui.theme
 
 import androidx.compose.foundation.isSystemInDarkTheme
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.darkColorScheme
 import androidx.compose.material3.lightColorScheme
 import androidx.compose.runtime.Composable
+import com.example.compose.md_theme_dark_background
```

Comment:
Do we need all those imports? Also, the package name is `com.example`.

Reply:
It was autogenerated by [composer](https://m3.material.io/theme-builder#/custom), as well as the package com.example

Reply:
Also i dont know how to chnage it back to com.palone yet